README.md

```diff
@@ -1 +1,12 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### This is Ram M Kripa's Homework 1 submission.
+I think I understand the **github workflow** for the most part now.
+I have used the *second option* listed on the course website (Local installation) because I already have a decent idea of how to use Github.
+I am a rising ***sophomore*** at UC Berkeley, currently planning to major in Engineering Mathematics and Statistics. I am really looking forward to the rest of this course. I primarily do research in Jupyter Notebooks, using Ipython. However, I would like to use R for research and for more Data Analysis projects in the future! My other interests include comics, cooking, and coffee! 
+> I also really like the 1966 Batman Live action TV show. *Adam West* is a legend.
+
+![The scourge of Gotham City](batman.jpg)
+
+## Some Days you just can't get rid of a Bomb
+![](bomb.jpg)
```

Comment:
Nice pictures! I like how you have labels that go with the pictures, a suggestion would be to perhaps make both labels uniform so use either the header (##) or the (>) block code for both picture descriptions. It may also be fun to link to a video from your favorite episode!